### PR TITLE
feat(core): add custom fields to atom feed

### PR DESCRIPTION
**Pull request by wb14123**
Atom allows to extend the xml with custom namespaces. This commit is
a first commit that tries to add some custom RSSHub fields to atom feed.

In this commit, I added three very common fields: upvotes, downvotes
and comments. Lots of websites has these common fields, like Youtube,
Twitter, Reddit, HackerNews and so on. Within these fields, RSS readers
can have more information to sort the contents, and can display these
information to user in a better format.

The document has been updated to reflect these new fields. I also changed
Bilibili video route as an example to show how to add number of comments
to the generated feed.

## 完整路由地址 / Example for the proposed route(s)

```routes
NOROUTE
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by wb14123** [comment]
Hi @TonyRL any comment on this proposal?

File: docs/joinus/quick-start.md [hunk 1]
```diff
@@ -318,6 +318,9 @@ ctx.state.data = {
             pubDate: '', // 文章发布时间
             guid: '', // 文章唯一标示, 必须唯一, 可选, 默认为文章链接
             link: '', // 指向文章的链接
+            upvotes: 0, // 默认为空，文章有多少 upvote
+            downvotes: 0, // 默认为空，文章有多少 downvote
+            comment: 0, // 默认为空，文章有多少评论
```
Comment (TonyRL) [inline docs/joinus/quick-start.md:323]: Please also update en docs

Comment (wb14123) [reply docs/joinus/quick-start.md]: @TonyRL updated en docs

File: docs/en/joinus/quick-start.md [hunk 2]
```diff
@@ -317,6 +317,9 @@ ctx.state.data = {
             pubDate: '', // The article publishing datetime
             guid: '', // The article unique identifier, optional, default to the article link below
             link: '', // The article link
+            upvotes: 0, // default to undefined, how many upvotes for this article,
+            downvotes: 0, // default to undefined, how many downvotes for this article,
+            comments: 0, // default to undefined, how many comments for this article
```
Comment (TonyRL) [inline docs/en/joinus/quick-start.md:322]: Actually, these fields should be mentioned in a separate section (e.g., like `Podcast feed` or `BT/Magnet feed` below).

They shouldn't be placed in `Produce RSS Feed` as new developers may think these are mandatory for an RSS feed.

**Pull request merged**